### DISCUSSION TITLE
How to set wifi SSID and password?

### DISCUSSION THREAD

**Discussion by hacstest04f** [gtxaspec/wz_mini_hacks#255] [Q&A] [answered]
Hello.  When first setting up the microSD card, how do you configure your home's wifi SSID and password?  I expected it to be inside the wz_mini.conf file, but that doesn't seem to be the case.  Does this boot up with its own wifi SSID and the you connect to it to set the wifi SSID and credentials?

Thanks for making this.

**Comment by gtxaspec** [comment, marked as answer]
@hacstest04f you need to set the wifi up using the phone app.

**Reply by hacstest04f** [reply]
Oh, thank you.  I didn't realize that it's still running the wyze firmware, just with additional functionality.

**Reply by edestecd** [reply]
Is it a requirement to setup with the app first? Or can I load the SD card on first boot after unboxing the camera and skip making a wyze account etc?

**Comment by iansmith42** [comment]
Can the WIFi settings be changed via ssh once the camera has been configured?  I have several APs with the same name for roaming and the cameras keep picking the ones far away to connect to. Would be nice to be able to select a specific SSID.

**Reply by virmaior** [reply]
wifi settings can be changed.

they are in /configs as 
```
/configs/.wifissid
/configs/.wifipasswd
```

be careful in that folder ...

**Reply by endertable** [reply]
> Can the WIFi settings be changed via ssh once the camera has been configured? I have several APs with the same name for roaming and the cameras keep picking the ones far away to connect to. Would be nice to be able to select a specific SSID.

Sounds like you have a mesh system.  The file you are probably wanting to look at is the /tmp/wpa_supplicant.conf file and probably specifying a BSSID.  The Wyze infrastructure creates that file on startup so you would need to create a custom script to update the file, then stop/restart the network using your wpa_supplicant.conf file.  

Just note, doing these things can lock you out of your camera if you don't create test/backout environment in your script to test all this first.  (of course, "lock out" not really true because you can easily remove and modify the SD card contents within wz_mini, so let's say more of a nuisance.)

**Reply by stephenmetzger** [reply]
Do you have a script for this?

**Comment by hacstest04f** [comment]
Is it possible for wz_mini_hacks would one day be able to config the wifi without having to use the Wyze app to configure the wifi first?  So if you had a brand new Wyze v3 cam, you don't need to ever have it touch wyze server at all?

**Reply by mach7** [reply]
I'm also interested in this idea @gtxaspec

**Reply by edestecd** [reply]
check out #144
